x-pack/filebeat/module/crowdstrike/falcon_audit/config/pipeline.js

```diff
@@ -0,0 +1,145 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+var crowdstrikeFalcon = (function() {
+    var processor = require("processor");
+
+    var convertUnderscore = function(text) {
+        return text.split(/(?=[A-Z])/).join('_').toLowerCase(); 
+    };
+
+    var decodeJson = new processor.DecodeJSONFields({
+        fields: ["message"],
+        target: "crowdstrike",
+        process_array: true,
+        max_depth: 8
+    });
+
+    // Don't process non-audit related events handled by the endpoint fileset
+    // This avoids generating duplicate events since both event types come from 
+    // the same log file. Abort early to minimize CPU time.
+    var cancelNonAuditEvents = function(evt) {
+        var eventType = evt.Get("crowdstrike.metadata.eventType")
+        switch (eventType) {
+            case "DetectionSummaryEvent":
+            case "IncidentSummaryEvent":
+                evt.Cancel()
+                break;
+        }  
+    };
+
+    var dropFields = function(evt) {
+        evt.Delete("message");
+    };
+  
+    var setFields = function (evt) {
+        evt.Put("agent.name", "falcon");
+    };
+
+    var convertFields = new processor.Convert({
+        fields: [
+            // UserActivityAuditEvent and AuthActivityAuditEvent
+            { from: "crowdstrike.event.UserIp", to: "source.ip", type: "ip" },
+        ],
+        mode: "copy",
+        ignore_missing: true,
+        ignore_failure: true
+    });
+
+    var parseTimestamp = new processor.Timestamp({
+        field: "crowdstrike.metadata.eventCreationTime",
+        target_field: "@timestamp",
+        timezone: "UTC",
+        layouts: ["UNIX_MS"],
+        ignore_missing: false,
+    });
+
+    var processEvent = function(evt) {
+        var eventType = evt.Get("crowdstrike.metadata.eventType")
+        var outcome = evt.Get("crowdstrike.event.Success")
+
+        evt.Put("event.kind", "event")
+
+        if (outcome == true) {
+            evt.Put("event.outcome", "success")
+        }
+        else if (outcome == false) {
```

Comment:
```suggestion
        else if (outcome === false) {
```

I think a strict-equality check would be better in most cases. Consider the case where `outcome = ""`, you'd get `failure` rather than `unknown`.